src/grammar.pegjs

```diff
@@ -52,6 +52,11 @@ Variable
   return [first].concat(rest).join('');
  }
 
+QuotedString
+ = _ '"' str:[ 0-9A-Za-z._@-]* '"' _ {
```

Comment:
It looks like parsing spaces works for double quotes. Do we want to support single quotes as well?

Reply:
I could go either way. It's not hard to add though. @rashidkpc thoughts?

Reply:
I'd consider adding support for single quotes as well since we support it in Canvas